appveyor.yml

```diff
@@ -0,0 +1,36 @@
+clone_depth: 5
+environment:
+  nodejs_version: "7.8"
+platform: x64 # flow needs 64b platforms
+# Install scripts. (runs after repo cloning)
+install:
+  # When Yarn 0.28 will be on Chocolatey, we'll be able to do the following to
+  # install latest version:
+  # install yarn and node using Chocolatey
+  #  - choco install yarn
+  # But until then we need to install manually
+  # 1. Node
+  - ps: Install-Product node $env:nodejs_version $env:platform
```

Comment:
x64 is needed for flow; we disable it for now but I'd rather still keep x64 here.